src/OrbitBase/ProcessIdLinux.cpp

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <filesystem>
+#include <optional>
+#include <system_error>
+
+#include "OrbitBase/Logging.h"
+#include "OrbitBase/ProcessId.h"
+#include "OrbitBase/Result.h"
+#include "absl/strings/numbers.h"
+
+namespace orbit_base {
+
+namespace fs = std::filesystem;
+
+namespace {
```

Comment:
not: I personally prefer static to anonymous namespaces but this is your call. (or we could have both also)

Reply:
I have a muscle memory to avoid static in these cases (I think the style guide aske for unnanmed namespace in the past). But I really don't care. Changed.